src/navigation/navigation.tsx

```diff
@@ -1,21 +1,58 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import { useDispatch } from 'react-redux'
+import { useAppSelector } from 'src/hooks'
```

Comment:
If we want use custom import for all hooks, lets do it for useDispatch, useEffect etc.

Reply:
I'm not sure if really need it, because our mobile project is not so difficult as web. I think we will store here only our custom hooks (such as useAppSelector).

Reply:
Sorry, but i don't understand why you customize this one hook "useSelector". How it differ from the rest?

Reply:
I have provided type `TypedUseSelectorHook<RootState>`. By default `useSelector` has `DefaultRootState` and it's not convenient for us. In order not to specify the type every time, I put this in `useAppSelector`. But if you really want I could put all our hooks in `hooks` folder.

Reply:
Thnx for your answer. I on the contrary want not to do unnecessary actions. And would not like to add customize import for other hooks.

Reply:
I agree. And I think we clarify this with Zhenya